src/common-lib/constants.ts

```diff
@@ -0,0 +1,7 @@
+export enum ENTRANCE {
+  ADMIN = 'admin-self-add',
+  LINK = 'magic-link',
+  ETH = 'manual-address-entry',
```

Comment:
let's rename `ETH` to something a bit more readable, how about
```suggestion
  MANUAL = 'manual-address-entry',
```